js/data.js

```diff
@@ -6,3 +6,16 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+// Saving to local storage
```

Comment:
This can be removed. It's also not factually correct, you only save to localStorage on line 16.